src/store/TranscriptionsStore.js

```diff
@@ -91,7 +91,7 @@ const TranscriptionsStore = types.model('TranscriptionsStore', {
   }
 
   const checkIfLocked = flow(function * checkIfLocked() {
-    const client = getRoot(self).client.tove
+    const client = getRoot(self).client
```

Comment:
You can use a destructured syntax here and the other similar lines in this same file: `const { client } = getRoot(self)`